src/ScrollSync.js

```diff
@@ -82,11 +87,11 @@ export default class ScrollSync extends Component {
         const paneHeight = pane.scrollHeight - clientHeight
         const paneWidth = pane.scrollWidth - clientWidth
         /* Adjust the scrollTop position of it accordingly */
-        if (vertical) {
-          pane.scrollTop = proportional ? (paneHeight * scrollTop) / (scrollHeight - clientHeight) : scrollTop // eslint-disable-line
+        if (vertical && scrollTopOffset) {
```

Comment:
I think `&& scrollOffset > 0` will be more readable and less ambiguous or what this check is doing?